src/adapters/bam/llm.ts

```diff
@@ -221,6 +223,29 @@ export class BAMLLM extends LLM<BAMLLMOutput, BAMLLMGenerateOptions> {
     }
   }
 
+  async embed(input: LLMInput[], options?: EmbeddingOptions): Promise<EmbeddingOutput> {
+    const max_embedding_inputs = 20;
```

Comment:
Use `camelCase` in JavaScript

Reply:
ooops.... updated https://github.com/i-am-bee/bee-agent-framework/pull/210/commits/21031e5bde5197ded616873e5c51fb7c11698812